services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/ProtectedPathSpec.groovy

```diff
@@ -111,9 +114,7 @@ class ProtectedPathSpec extends BaseSpecification {
                 def rules = northbound.getSwitchRules(sw.switchId).flowEntries.findAll {
                     !Cookie.isDefaultRule(it.cookie)
                 }
-
-                assert flowHelper.findForwardPathRules(rules, sw).size() == 0
-                assert flowHelper.findReversePathRules(rules, sw).size() == 0
+                assert rules.any { it != protectedForwardCookie || it != protectedReverseCookie }
```

Comment:
Hold for a second and try to think of what you are verifying here

Reply:
oh, I  should you 'AND' instead of 'OR'
thx, fixed

Reply:
should be `every`